docs/core/tools/project-json-to-csproj.md

```diff
@@ -0,0 +1,592 @@
+
+## tools
+```json
+{
+  "tools": {
+    "Microsoft.EntityFrameworkCore.Tools.DotNet": "1.0.0-*"
+  }
+}
+```
+
+```xml
+<ItemGroup>
+  <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.0.0" />
+</ItemGroup>
+```
+
+> [!NOTE]
+> `imports` on tools are not supported in csproj. Tools that need imports will not work with
+the new `Microsoft.NET.Sdk`.
+
+## buildOptions
+
+See also [Files](#files).
+
+### emitEntryPoint
+
+```json
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <OutputType>Exe</OutputType>
+</PropertyGroup>
+```
+
+If `emitEntryPoint` was `false`, the value of `OutputType` is converted to `Library`, which is the default value:
+
+```json
+{
+  "buildOptions": {
+    "emitEntryPoint": false
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <OutputType>Library</OutputType>
+  <!-- or, omit altogether. It defaults to 'Library' -->
+</PropertyGroup>
+```
+
+### keyFile
+
+```json
+{
+  "buildOptions": {
+    "keyFile": "MyKey.snk"
+  }
+}
+```
+
+The `keyFile` element expands to three properties in MSBuild:
+
+```xml
+<PropertyGroup>
+  <AssemblyOriginatorKeyFile>MyKey.snk</AssemblyOriginatorKeyFile>
+  <SignAssembly>true</SignAssembly>
+  <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
+</PropertyGroup>
+```
+
+### Other common build options
+
+```json
+{
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "nowarn": ["CS0168", "CS0219"],
+    "xmlDoc": true,
+    "preserveCompilationContext": true,
+    "outputName": "Different.AssemblyName",
+    "debugType": "portable",
+    "allowUnsafe": true,
+    "define": ["TEST", "OTHERCONDITION"]
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  <NoWarn>$(NoWarn);CS0168;CS0219</NoWarn>
+  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  <PreserveCompilationContext>true</PreserveCompilationContext>
+  <AssemblyName>Different.AssemblyName</AssemblyName>
+  <DebugType>portable</DebugType>
+  <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  <DefineConstants>$(DefineConstants);TEST;OTHERCONDITION</DefineConstants>
+</PropertyGroup>
+```
+
+## packOptions
+
+See also [Files](#files).
+
+### Common pack options
+```json
+{
+  "packOptions": {
+    "summary": "A bundle of cats",
+    "tags": ["hyperscale", "cats"],
+    "owners": [ "Nate", "Jenna" ],
+    "releaseNotes": "Version 1.0",
+    "iconUrl": "https://icons.com/awesomeness.png",
+    "projectUrl": "https://github.com/natemcmaster",
+    "licenseUrl": "https://www.apache.org/licenses/LICENSE-2.0",
+    "requireLicenseAcceptance": false,
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/natemcmaster/natemcmaster.github.io"
+    },
+    "owners": ["Fabrikam", "Microsoft"]
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <Summary>A bundle of cats</Summary>
```

Comment:
To clarify, NuGet doesn't support `<Summary>`. PJ's packOptions/summary doesn't have an equivalent in csproj. 

csproj added a new field, `<Title>`, but this is not the same as packOptions/summary.